babel.config.js

```diff
@@ -0,0 +1,7 @@
+// babel.config.js
+module.exports = {
```

Comment:
I'm curious what jest needs Babel for? I'm using jest on other projects without Babel so I'm curious what purpose it has. Is this related to the recent jest port to TypeScript? (Think I'm still using the older one)

Reply:
Jest can either run typescript tests by directly running them with `ts-jest` (which afaik is like `ts-node`) or by transpiling them with babel (because afaik jest uses babel anyways to make modern syntax available with old node versions).

In this case, I tried ts-jest at first, but since some tests are mocking the `typescript` import, that wasn't possible because that would break ts-jest.